core/object/script_language.cpp

```diff
@@ -43,7 +43,7 @@ int ScriptServer::_language_count = 0;
 
 bool ScriptServer::scripting_enabled = true;
 bool ScriptServer::reload_scripts_on_save = false;
-bool ScriptServer::languages_finished = false;
+std::atomic<bool> ScriptServer::languages_finished = false;
```

Comment:
```suggestion
SafeFlag ScriptServer::languages_finished; // Used until GH-76581 is fixed properly.
```

(In Godot, we're not using `<atomic>` directly. There's the `SafeFlag` Godotic wrapper that you should use instead.)